target/xilinx/flavor_vanilla/flavor_vanilla.mk

```diff
@@ -46,6 +46,7 @@ $(CAR_XIL_DIR)/flavor_vanilla/scripts/add_sources.tcl: Bender.yml
 
 # Compile bitstream
 $(CAR_XIL_DIR)/flavor_vanilla/out/%.bit: $(xilinx_ips_paths_vanilla) $(CAR_XIL_DIR)/flavor_vanilla/scripts/add_sources.tcl
+	@if [ "$(GEN_NO_HYPERBUS)" != "1" ] && [ "$(XILINX_ELABORATION_ONLY)" != "1" ]; then echo "Hyperbus not supported yet in this branch"; fi;
```

Comment:
what does "[...] not supported yet in this branch" mean? Since this would land in main, I guess this is a leftover of your dev branch? I think the echo message is unclear and could be improved for better clarity